concepts/Common/Integration Guides/60 Manage Amazon S3 Storage with DevExtreme Components/10 Amazon Setup.md

```diff
@@ -0,0 +1,38 @@
+### Create a New Bucket
+
+**Main article**: [Amazon Documentation — Creating a Bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html)
```

Comment:
It seems strange that you link the reader directly to the AWS site at the beginning of your topic section.
To keep readers engaged, wouldn't it be better to place this at the end of the section so that users can click it to find further details?